python/selfie-lib/selfie_lib/SelfieImplementations.py

```diff
@@ -140,12 +147,16 @@ def __actual(self) -> str:
     def to_be_TODO(self, unused_arg: Any = None) -> str:
         return _toBeDidntMatch(None, self.__actual(), LiteralString())
 
-    def to_be(self, expected: str) -> str:
+    def to_be(self, expected: Union[str, int, bool]) -> str:
         actual_string = self.__actual()
         if actual_string == expected:
             return _checkSrc(actual_string)
         else:
-            return _toBeDidntMatch(expected, actual_string, LiteralString())
+            return _toBeDidntMatch(
+                expected if isinstance(expected, str) else None,
```

Comment:
Passing None here is risky. Better to disable type checking for this line, or throw an exception earlier if the arg isn't a string.